### PR TITLE
+doc more visible note on connection pool shutdown

### PR DESCRIPTION
**Pull request by ktoso**
This should help a bit in people noticing this, also makes it googlable by `AbruptTerminationException`

### CODE REVIEW

**Comment by johanandren** [comment]
LGTM

File: akka-docs/rst/java/http/client-side/host-level.rst [hunk 1]
```diff
@@ -144,6 +144,10 @@ It's also possible to trigger the immediate termination of *all* connection pool
 time by calling ``Http.get(system).shutdownAllConnectionPools()``. This call too produces a ``CompletionStage<Done>`` which is fulfilled when
 all pools have terminated.
 
+.. note::
+  When encoutering unexpected ``akka.stream.AbruptTerminationException`` exceptions during ``ActorSystem`` **shutdown**
+  please make sure that active connections are shut down before shutting down the entire system, this can be done by
+  calling the ``Http().shutdownAllConnectionPools()`` method, and only once its Future completes, shutting down the actor system.
```
Comment (patriknw) [inline akka-docs/rst/java/http/client-side/host-level.rst]: is it Future or CompletationStage

`Http()` doesn't look javaish

**Comment by patriknw** [comment]
LGTM after nitpicks

**Comment by ktoso** [comment]
Thanks for catching the nitpicks, very true. Will fix

File: akka-docs/rst/java/http/client-side/host-level.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/java/http/client-side/host-level.rst]: fixed

**Pull request merged**